### ISSUE TITLE
How to use demo.py to get the visual feature of each region (box)?

### ISSUE THREAD

**Issue by HaoYang0123** [jwyang/faster-rcnn.pytorch#778]
I download VGG-16 model and use this model in demo.py. It returns multiple boxes in each image, however it doesn't output the visual features of these boxes. How to modify this code?

**Comment by loolzaaa** [comment]
> it doesn't output the **visual features** of these boxes

What do you mean? Network visualize bounding boxes on image, class name and probability.

**Comment by HaoYang0123** [comment]
I have resolved this problem. I need each region feature, you can find in lib/model/faster_rcnn/faster_rcnn.py (line 86): pooled_feat = self._head_to_tail(pooled_feat). 
I need pooled_feat, if I use resnet-101 model, pooled_feat shape is 300*2048, which has 300 regions and each region has features (dim=2048).

**Comment by badman7** [comment]
Is your torch.utils.ffi running correctly?

**Comment by KrystalCWT** [comment]
> I have resolved this problem. I need each region feature, you can find in lib/model/faster_rcnn/faster_rcnn.py (line 86): pooled_feat = self._head_to_tail(pooled_feat). I need pooled_feat, if I use resnet-101 model, pooled_feat shape is 300*2048, which has 300 regions and each region has features (dim=2048).

HI, I have  the same problem. Can you share your code?
